stresstest-aws/bootstrap.tf

```diff
@@ -64,18 +78,37 @@ resource "local_file" "genesis-file" {
       "eip155Block": 0,
       "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
       "eip158Block": 0,
+%{if var.native_geth == false~}
       "isQuorum": true,
+%{endif~}
 %{if var.consensus == "ibft"~}
       "istanbul": {
         "epoch": 30000,
         "policy": 0,
         "ceil2Nby3Block": 0
       },
+%{endif~}
+%{if var.consensus == "clique"~}
+      "clique": {
+        "epoch": 30000,
+        "period": ${var.blockPeriod}
+      },
 %{endif~}
       "maxCodeSize": 50
     },
     "difficulty": "${var.consensus == "ibft" ? "0x1" : "0x0"}",
-    "extraData": "${var.consensus == "ibft" ? quorum_bootstrap_istanbul_extradata.this.extradata : "0x0000000000000000000000000000000000000000000000000000000000000000"}",
+%{if var.consensus == "ibft"~}
+    "extraData": "${quorum_bootstrap_istanbul_extradata.this.extradata}",
+%{endif~}
+
+%{if var.consensus == "clique"~}
+    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000${local.clique_extra_data}0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
please move the full extra data value to `local.clique_extra_data` variable so we just need to reference here as:
```
...
 "extraData": "${local.clique_extra_data}",
...
```
It provides the better understanding of `local.clique_extra_data`.